lua/cspell/code_actions/make_add_to_dictionary_action.lua

```diff
@@ -23,7 +23,13 @@ return function(opts)
     local misspelled_word = opts.diagnostic.user_data.misspelled
 
     return {
-        title = 'Add "' .. misspelled_word .. '" to dictionary "' .. opts.dictionary.name .. '"',
+        title = 'Add "'
+            .. misspelled_word
+            .. '" to dictionary "'
+            .. opts.dictionary.name
+            .. '"['
+            .. opts.cspell.path
+            .. "]",
```

Comment:
The formatting for this title needs some work, the path can be super long for a message and it has a leading double quote we don't need.

We should try compressing the path into a relative path when possible, or replace `vim.env.HOME` with `~` for brevity's sake.

Something like: `Add "${word}" to dictionary "${dictionary}" from ${short_config_path}`

Where short_config_path can be something like:

```lua
local short_config_path = Path:new(opts.cspell.path)
    :expand()
    :gsub(Path:new("."):expand(), ".") -- make it relative, if possibile
    :gsub(vim.env.HOME, "~") -- replace the home path with the alias
```

It could be a helper function to reuse in the other action and in the spec.

Reply:
Added the helper function in the new commit